src/components/slider/handle.js

```diff
@@ -24,33 +24,39 @@ class Handle extends Component {
       getHandleProps,
     } = this.props
 
+    const btnStyle = {
+      height: 14,
+      width: 14,
+      left: `${percent}%`,
+      position: 'absolute',
+      transform: 'translate(-50%, -50%)',
+      WebkitTapHighlightColor: 'rgba(0,0,0,0)',
+      border: 0,
+      borderRadius: '50%',
+      backgroundColor: disabled ? 'transparent' : 'var(--slider-track-bg)',
+      zIndex: 1,
+    }
+
     const { mouseOver } = this.state
 
     return (
-      <button
-        role="slider"
-        aria-valuemin={min}
-        aria-valuemax={max}
-        aria-valuenow={value}
-        style={{
-          height: 14,
-          width: 14,
-          left: `${percent}%`,
-          position: 'absolute',
-          transform: 'translate(-50%, -50%)',
-          WebkitTapHighlightColor: 'rgba(0,0,0,0)',
-          border: 0,
-          borderRadius: '50%',
-          backgroundColor: disabled ? 'transparent' : 'var(--slider-track-bg)',
-          zIndex: 1,
-        }}
-        {...getHandleProps(id, {
-          onMouseEnter: this.onMouseEnter,
-          onMouseLeave: this.onMouseLeave,
-        })}
+      <Tooltip
+        text={value}
+        style={{ fontWeigth: 700, fontSize: '14px' }}
```

Comment:
The `tooltip` should follow the handler position, not sure if this is possible with this component since it auto-detect / assigns the position ?

Also it flicking when dragging and not aligned properly.

Reply:
Looks like it follows but when I do the first click is not aligned on the same position.
Any ideas ?

Reply:
I guess we should use another component, please revert this changes to use the previous `slider-tooltip` component, unless you find a solution.